### DISCUSSION TITLE
3 websites that use WalletConnect are not listed on you website. Are they safe ?

### DISCUSSION THREAD

**Discussion by NotMainstream** [WalletConnect/walletconnect-monorepo#2067] [Developer Support]
https://apphex.win/!/  ; 
https://go.tshare.app/  ;  
https://hexmob.win/ 
These websites have WalletConnect but they are not listed on your website were you list 16 pages of Dapps. 
Please confirm if they are safe for staking HEX token.
Please also let us know if there are any other websites for staking HEX token that are using WalletConnect.
I m intending of using a Ledger Nano S on an Android phone and I need WalletConnect to connect to the Dapps.

**Comment by finessevanes** [comment]
Hey - websites don't have to be listed on our explorer page to be legit. **However** we only vet that functions like signing transactions work on the dapp. Just beacuse they are listed on our explorer page does not mean we are confirming they are safe to use, just that the functionality works. 

Always do your own research anytime you're interacting with a dapp.
